### PR TITLE
Update keras cv docs to the newest release

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

File: scripts/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+tmp*
```
Comment (fchollet) [inline scripts/.gitignore:1]: Please revert this file (if we need a gitignore we can add a complete one in a separate PR)

File: scripts/autogen.py [hunk 2]
```diff
@@ -220,8 +220,13 @@ def process_one_dir(src_dir, target_dir):
 
     def add_example(self, path, working_dir=None):
         """e.g. add_example('vision/cats_and_dogs')"""
-        assert path.count(os.path.sep) == 1
+
+        # Prune out the ../ path
```
Comment (fchollet) [inline scripts/autogen.py:224]: Didn't we merge this change already? Consider rebasing

File: scripts/cv_api_master.py [hunk 3]
```diff
@@ -101,6 +106,40 @@
     ],
 }
 
+BOUNDING_BOX_FORMATS = {
+        "path": "formats",
```
Comment (fchollet) [inline scripts/cv_api_master.py:110]: Too much indentation

File: templates/keras_cv/bounding_box/index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# KerasCV Bounding Boxes
+All KerasCV components that process bounding boxes require a `bounding_box_format`
```
Comment (fchollet) [inline templates/keras_cv/bounding_box/index.md:2]: Add line break above

File: templates/keras_cv/bounding_box/index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+argument.  This argument allows you to seamlessly integrate KerasCV components into
+your own workflows while preserving proper behavior of the components themselves.
+
+[The list of bounding box formats supported in KerasCV are listed in the API docs](/api/keras_cv/bounding_box/formats).
```
Comment (fchollet) [inline templates/keras_cv/bounding_box/index.md:6]: `The bounding box formats supported in KerasCV [are listed in the API docs](/api/keras_cv/bounding_box/formats)`

File: templates/keras_cv/index.md [hunk 6]
```diff
@@ -25,6 +25,42 @@ pip install keras-cv --upgrade
 You can also check out other versions in our
 [GitHub repository](https://github.com/keras-team/keras-cv/releases).
 
+## Quick Introduction
+Create a preprocessing pipeline:
```
Comment (fchollet) [inline templates/keras_cv/index.md:29]: Add line break before and after

File: templates/keras_cv/index.md [hunk 7]
```diff
@@ -25,6 +25,42 @@ pip install keras-cv --upgrade
...
+```python
+import keras_cv
+import tensorflow as tf
```
Comment (fchollet) [inline templates/keras_cv/index.md:32]: In general if you just use `keras` use `from tensorflow import keras`

File: templates/keras_cv/index.md [hunk 8]
```diff
@@ -25,6 +25,42 @@ pip install keras-cv --upgrade
...
+densenet = keras_cv.models.DenseNet(
+  include_rescaling=True,
+  include_top=True,
+  num_classes=102
+)
+densenet.compile(optimizer='adam', metrics=['accuracy'])
+```
+
+Fit your model:
+```python
```
Comment (fchollet) [inline templates/keras_cv/index.md:60]: Add line break

File: templates/keras_cv/index.md [hunk 9]
```diff
@@ -25,6 +25,42 @@ pip install keras-cv --upgrade
...
+Augment a TensorFlow dataset:
+
+```python
+dataset = dataset.map(lambda images, labels: {"images": images, "labels": labels})
+dataset = dataset.map(preprocessing_model)
+dataset = dataset.map(lambda inputs: (inputs["images"], inputs["labels"]))
+```
+
+Create a model:
+```python
```
Comment (fchollet) [inline templates/keras_cv/index.md:50]: Add line break

File: templates/keras_cv/index.md [hunk 10]
```diff
@@ -25,6 +25,42 @@ pip install keras-cv --upgrade
...
+import tensorflow as tf
+
+preprocessing_model = tf.keras.Sequential([
+    keras_cv.layers.RandAugment(value_range=(0, 255))
+    keras_cv.layers.CutMix(),
+    keras_cv.layers.MixUp()
+], name="preprocessing_model")
+```
+
+Augment a TensorFlow dataset:
```
Comment (fchollet) [inline templates/keras_cv/index.md:41]: Consider using the full path to get the link, `tf.data.Dataset`

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: scripts/cv_api_master.py (hunk 3 above)
Comment (LukeWood) [reply scripts/cv_api_master.py:110]: Ran `black` on this file and it fixed it

Thanks!

**Comment by LukeWood** [comment]
@fchollet rebased and addressed all comments.  Thanks!

**Review by fchollet — APPROVED** [review]

**Comment by LukeWood** [comment]
@fchollet added the GenFiles!

**Comment by LukeWood** [comment]
@fchollet also added a models page

File: guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md [hunk 11]
```diff
@@ -182,9 +183,9 @@ visualize_dataset(train_dataset, title="After RandAugment")
 ```
 
 
-
-![png](/img/guides/cut_mix_mix_up_and_rand_augment/cut_mix_mix_up_and_rand_augment_15_0.png)
-
+    
+![png](../guides/img/cut_mix_mix_up_and_rand_augment/cut_mix_mix_up_and_rand_augment_15_0.png)
```
Comment (fchollet) [inline guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md:187]: I don't know how you generate the guides, but the image links have an incorrect path -- it should be `/img/guides/`.

File: guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md [hunk 12]
```diff
@@ -421,8 +415,15 @@ with strategy.scope():
 
 <div class="k-default-codeblock">
 ```
-32/32 [==============================] - ETA: 0s - loss: 4.8694 - accuracy: 0.0049
-32/32 [==============================] - 399s 12s/step - loss: 4.8694 - accuracy: 0.0049 - val_loss: 43.5487 - val_accuracy: 0.0187
+WARNING:tensorflow:There are non-GPU devices in `tf.distribute.Strategy`, not using nccl allreduce.
```
Comment (fchollet) [inline guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md:418]: Let's remove these warnings.

**Review by fchollet — COMMENTED** [review]
Thanks for the updates! We should fix the image links.

File: guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md (hunk 12 above)
Comment (LukeWood) [reply guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md:418]: Sounds good.  unfortunately I don’t think tf.distribute has an API to mute these

Comment (fchollet) [reply guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md:418]: That's too bad, but manual edits sound good.

File: guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md (hunk 11 above)
Comment (fchollet) [reply guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md:187]: Please manually change the links and then use `python autogen.py make` / `serve` to check that it worked. I'll do some local tests to see what's wrong with the generation afterwards...

Comment (LukeWood) [reply guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md:187]: We could always hard code a string replace operation in autogen too if we want to go the low tech route.

**Review by fchollet — APPROVED** [review]
LGTM, thanks! I'll take a look at fixing KerasCV guide generation going forward.
